app/src/main/java/space/taran/arknavigator/utils/ComparatorUtils.kt

```diff
@@ -16,3 +19,9 @@ fun <T> unequalCompareBy(comparator: Comparator<T>): Comparator<T> =
             )
         } else result
     }
+
+fun compareByScores(scoreStorage: ScoreStorage) = Comparator<ResourceItem> { a, b ->
```

Comment:
Better to move it somewhere closer to `ScoreStorage`, maybe even in the same module. Or to `ResourceGridPresenter` since it is used only there.